great_expectations/checkpoint/types/checkpoint_result.py

```diff
@@ -288,10 +288,10 @@ def _list_data_assets_validated_by_batch_id(self) -> dict:
                         "expectation_suite_names": [expectation_suite_name],
                     }
                 else:
-                    assets_validated_by_batch_id[batch_id]["validation_results"].append(
+                    assets_validated_by_batch_id[batch_id]["validation_results"].append(  # type: ignore[union-attr]
                         validation_result
                     )
-                    assets_validated_by_batch_id[batch_id]["expectation_suite_names"].append(
+                    assets_validated_by_batch_id[batch_id]["expectation_suite_names"].append(  # type: ignore[union-attr]
```

Comment:
This is the old checkpoint result (not sure why mypy is mad about it now). We're using a new Pydantic type so I think we can ignore these?